docs/usage/self-hosted-experimental.md

```diff
@@ -32,6 +32,25 @@ Skipping the check will speed things up, but may result in versions being return
 
 If set to any value, Renovate will always paginate requests to GitHub fully, instead of stopping after 10 pages.
 
+## `RENOVATE_X_DELETE_CONFIG_FILE`
+
+If `true` Renovate tries to delete the self-hosted config file after reading it.
+You can set the config file Renovate should read with the `RENOVATE_CONFIG_FILE` environment variable.
+
+The process that runs Renovate must have the correct permissions to delete the config file.
+
+## `RENOVATE_X_DOCKER_HUB_DISABLE_LABEL_LOOKUP`
+
+If set to any value, Renovate will skip attempting to get release labels (e.g. gitRef, sourceUrl) from manifest annotations for `https://index.docker.io`.
```

Comment:
```suggestion
If set to any value, Renovate will not fetch any release labels from manifest annotations.

For example, when set to any value, Renovate skips getting the `gitRef` and/or `sourceUrl` release labels from the `https://index.docker.io` manifest annotations.
```

Let's split the sentence into two parts. One sentence for the basic idea. And one sentence with the example bits.